noxfiles/docker_nox.py

```diff
@@ -26,11 +26,14 @@ def get_current_image() -> str:
 )
 def build(session: nox.Session, image: str) -> None:
     """Build the Docker container for fidesctl."""
+
+    # The lambdas are a workaround to lazily evaluate get_current_image
+    # This allows the dev deployment to run without needing other dev requirements
```

Comment:
oh, clever 🧠  you use lambdas more than anyone I know 😂

Reply:
sometimes its a necessary evil 😂